packages/hw-app-eth/src/Eth.ts

```diff
@@ -1201,43 +1236,52 @@ function provideNFTInformation(
 function setExternalPlugin(
   transport: Transport,
   payload: string,
-  signature: string
+  signature: string,
+  loadConfig: LoadConfig
 ): Promise<boolean> {
+  const { strictMode } = loadConfig;
   const payloadBuffer = Buffer.from(payload, "hex");
   const signatureBuffer = Buffer.from(signature, "hex");
   const buffer = Buffer.concat([payloadBuffer, signatureBuffer]);
+
   return transport.send(0xe0, 0x12, 0x00, 0x00, buffer).then(
     () => true,
     (e) => {
-      if (e && e.statusCode === 0x6a80) {
+      if (e?.statusCode === 0x6a80) {
         // this case happen when the plugin name is too short or too long
-        return false;
-      } else if (e && e.statusCode === 0x6984) {
+        return strictModeHandler(strictMode, e);
+      } else if (e?.statusCode === 0x6984) {
         // this case happen when the plugin requested is not installed on the device
```

Comment:
in this case, there is a possibility, even inside Ledger Live, that you enter this case and it's actually "legit". Example: contract that accept funds in and would be plugin. it's a edge case, but it's a possibility: users trying to do a regular send eth to a contract and didn't installed the possible plugin that relates to it would be forced to install it. in that case we don't seem to offer a clear error so i don't think we want to do it like this, we probably want to have specific error because the case is "legit" to not have installed the plugin and we may want to drive to a specific wording / clear error, on this case.
we need to catch up with someone knowing deeply the app and its edge cases.